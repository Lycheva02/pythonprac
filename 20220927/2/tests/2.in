373, 0, 919892389, 35, 704, 919 
