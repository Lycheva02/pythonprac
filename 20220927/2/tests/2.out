[0, 704, 919892389, 35, 373, 919]
